ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml

```diff
@@ -2680,9 +2680,20 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-ibmcloud-ipi-private-fips
-- as: nutanix-ipi-p2-f3
+- as: nutanix-ipi-boot-categories-project-p2-f7
   cluster: build01
-  cron: 4 19 3,7,10,13,16,19,22,25,28,31 * *
+  cron: 55 11 1,8,15,23 * *
```

Comment:
Hmm, I tried to generate the cron locally, but seem like get different value with yours.
```
$ generate-cron-entry.sh nutanix-ipi-boot-categories-project-p2-f7 openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml
cron: 15 14 6,14,22,29 * *
```